fixtures/u2_import_subsymbols_namespaced.js.sem.uast

```diff
@@ -152,6 +152,68 @@
             },
             Target: ~,
          },
+         { '@type': "uast:Import",
+            '@pos': { '@type': "uast:Positions",
+               start: { '@type': "uast:Position",
+                  offset: 65,
+                  line: 3,
+                  col: 1,
+               },
+               end: { '@type': "uast:Position",
+                  offset: 108,
+                  line: 3,
+                  col: 44,
+               },
+            },
+            All: true,
```

Comment:
This is actually incorrect and was the reason why I think we should rewrite the normalizer for imports.

Still, it may be okay to merge this and then work on a proper fix.

Reply:
Thank you for pointing it out - yes, I noticed something fishy is going on with this [All flag](https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast#Import).

Let me look into it again - but in this case, help me understand, you mean that a single type should always be `False`? That way this flag would _only_ be `True` for whiledcard imports. 
Is that correct?

Reply:
Correct, `All` is true only for wildcards. For this case, it should be false, and `Names` should ideally include the name of the type that is imported.

Reply:
added as TODO in 7b65c80